CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ typed as `MessageLabel` and are now regular `UILabel`s.
 - All `DetectorType`s for `MessageLabel` are disabled by default.
 [#356](https://github.com/MessageKit/MessageKit/pull/356) by [@SD10](https://github.com/sd10).
 
+- **Breaking Change** Changed the name of  `MessageInputBar`'s property `maxHeight` to `maxTextViewHeight` as the property is the max height the `InputTextView` can have, not the `MessageInputBar` itself.
```

Comment:
This is added under the last release, we need to move this into `Upcoming Release`